libs/shared/typescript/charts-angular/project.json

```
@@ -5,6 +5,13 @@
   "sourceRoot": "libs/shared/typescript/charts-angular/src",
   "prefix": "sage",
   "targets": {
+    "test": {
+      "executor": "@nx/jest:jest",
+      "outputs": ["{workspaceRoot}/coverage/libs/shared/typescript/charts-angular"],
```

Using the variable `{projectRoot}` makes the task more portable, e.g. when copy-pasting a `project.json`. `{projectRoot}` resolves to the path of project folder starting from the workspace root.

```suggestion
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
```